website/src/models/group.js

```diff
@@ -151,6 +151,50 @@ schema.statics.getGroup = async function getGroup (options = {}) {
   return group;
 };
 
+schema.statics.getGroups = async function getGroups (options = {}) {
+  let {user, types, groupFields = basicFields, sort = '-memberCount', populateLeader = false} = options;
+  let queries = [];
+
+  types.forEach(type => {
```

Comment:
but it seems that we've been doing the same thing when getting all groups so using different queries is probably okay for now since this route won't be used much